app/src/main/java/org/simple/clinic/home/patients/PatientsTabUiRenderer.kt

```diff
@@ -22,16 +23,14 @@ class PatientsTabUiRenderer(
       showAccountNotifications(model)
     }
 
-    if (model.hasLoadedNumberOfPatientsRegistered) {
-      toggleTrainingViewVisibility(model)
-    }
+    val isDrugStockReportFilledOrNotPresent = model.isDrugStockReportFilled == true || !model.isDrugStockReportFilled.toOptional().isPresent
```

Comment:
Instead of converting the nullable back to `Optional`. Could we have used something like this?

```kt
val isDrugStockReportFilledOrNotPresent = model.isDrugStockReportFilled == null || model.isDrugStockReportFilled
```

This way Kotlin compiler will also do the smart casting of the boolean. WDYT?